PowerPointLabs/PowerPointLabs/ELearningLab/Utility/ShapeUtility.cs

```diff
@@ -30,24 +30,32 @@ public static Shape InsertDefaultCalloutBoxToSlide(PowerPointSlide slide, string
         {
             float slideWidth = PowerPointPresentation.Current.SlideWidth;
             float slideHeight = PowerPointPresentation.Current.SlideHeight;
-           
+
             Shape calloutBox = slide.Shapes.AddShape(MsoAutoShapeType.msoShapeRoundedRectangularCallout, 10, 10, 100, 10);
             calloutBox.Name = shapeName;
             calloutBox.TextFrame.TextRange.Text = calloutText;
+            calloutBox.Left = 10;
+            calloutBox.Top = 10;
             calloutBox.TextFrame.AutoSize = PpAutoSize.ppAutoSizeShapeToFitText;
-            calloutBox.TextFrame.WordWrap = MsoTriState.msoTrue;
             calloutBox.TextEffect.Alignment = MsoTextEffectAlignment.msoTextEffectAlignmentLeft;
+            FormatCalloutToDefaultStyle(calloutBox);
+
+            return calloutBox;
+        }
+
+        /// <summary>
+        /// Formats shape to default style.
+        /// </summary>
+        public static void FormatCalloutToDefaultStyle(Shape calloutBox)
```

Comment:
Should we put this into `ShapeUtil` or even into `TooltipsLab`? It seems weird to import something from eLearningLab.

Reply:
I am of the opinion that we can move this to `ShapeUtil` as a default callout styling method

Reply:
@blewjy @ChesterSng Should it be a separate function that ELL does not use?

Reply:
`FormatCalloutToDefaultStyle` is not used by ELL currently right?

Reply:
If we are moving it to `ShapeUtil`, I think it should be a method used by both ELL and tooltipsLab. 

Otherwise, if we want to separate them completely, we can just duplicate the formatting in TooltipsLab and don't modify ELL at all.

Reply:
I think shifting it to `ShapeUtil` will be a better decision from a SE pov, since we can reuse it for other labs if needed next time with this formatting. @LiXiaoooowei are you okay with us refactoring some code in ELL to `ShapeUtil` so TooltipsLab can use it too?

Reply:
@ChesterSng Yes I am ok with that. Since both eLL and ToolTipsLab use the same function, it should be moved to `ShapeUtil`.